models/activities.js

```diff
@@ -136,7 +136,7 @@ if (Meteor.isServer) {
         params.board = '';
       }
       title = 'act-withBoardTitle';
-      params.url = board.absoluteUrl();
+      params.url = board.originRelativeUrl();
```

Comment:
Should we use `absoluteUrl()` here instead ?

Reply:
@Majed6 

Please try to get it working everywhere with relative URLs.

Reply:
I've reverted this. A relative notification (email) doesn't make a loot of sense. Relative to whom ? The activity maker or the activity notification receiver ?